proto/mixer.proto

```diff
@@ -610,13 +618,13 @@ message GetStatSeriesRequest {
   // dcid of the stat var.
   string stat_var = 2;
   // (optional) measurement method of the observation, ex: "CensusACS5yrSurvey".
-  string measurement_method = 4;
+  string measurement_method = 3;
   // (optional) observation period of the observation, ex: "P1Y".
-  string observation_period = 5;
+  string observation_period = 4;
   // (optional) unit of the observation.
-  string unit = 6;
+  string unit = 5;
   // (optional) scaling factor of the observation.
-  string scaling_factor = 7;
+  string scaling_factor = 6;
```

Comment:
Is it better practice to have the GetStatValue and GetStatSeries Request be duped rather than combined?

Reply:
yeah, usually each A service will have corresponding ARequest and AResponse as req and resp message names.